scripts/runProduction.sh

```diff
@@ -0,0 +1,2 @@
+echo "Running Production Site"
+./build.js -run --settings settings.json
```

Comment:
This script appears misnamed. It should at least pass --production, but probably also should be a different settings.json and have a mongo URL.